beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/service/BeekeeperService.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2019-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.api.service;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Service;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+import com.expediagroup.beekeeper.core.repository.HousekeepingMetadataRepository;
+
+@Service
+public class BeekeeperService implements HousekeepingEntityService<HousekeepingMetadata>{
```

Comment:
Is this `BeekeeperService` just going to be used for metadata? If so then probably want to rename is to something else. Maybe `BeekeeperHousekeepingMetadataService`

Reply:
It's going to be used for everything, so I've kept the name general

Reply:
`Beekeeper` is the name of the app/component. I'd suggest coming up with a better name for it. `BeekeeperService` runs the risk of becoming a _catch all_ service where all the methods are added.